5calls/app/src/main/java/org/a5calls/android/a5calls/controller/StatsActivity.java

```diff
@@ -255,10 +384,7 @@ private void sendShare() {
         shareIntent.putExtra(Intent.EXTRA_TEXT,
                 String.format(getResources().getString(R.string.share_content), mCallCount));
         shareIntent.setDataAndType(imageUri, "image/png");
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
-            // Needed to avoid security exception on KitKat.
-            shareIntent.setClipData(ClipData.newRawUri(null, imageUri));
-        }
+        shareIntent.putExtra(Intent.EXTRA_STREAM, imageUri);
```

Comment:
Does this work on Kitkat? (You can try with an emulator if you don't have a device.)

Reply:
Tried this on 4.4.2 tablet right now and the image appeared in the message. Development was done on 6.0.1 phone.